src/viking.js

```diff
@@ -46,7 +46,13 @@ class Viking extends Soldier  {
 
 // Saxon
 class Saxon extends Soldier {
-
+  receiveDamage(theDamage){
+    this.health = this.health - theDamage;
+    if (health > 0)
+      return 'A Saxon has received ${theDamage} points of damage';
+    else
+     return 'A Saxon has died in act of combat';
+  }
```

Comment:
- On line 51, you forgot keyword `this` -> `this.health`
- On line 52, if you're using string concatenation (inserting js variables into string), you must use backticks
- On line 54 , again syntax error..